pkg/plugin/view/vulnerabilities/report_view.go

```diff
@@ -86,7 +86,7 @@ func NewReport(workload kube.Object, vulnerabilityReportsDefined bool, reports [
 
 		items = append(items, component.FlexLayoutItem{
 			Width: component.WidthFull,
-			View:  createVulnerabilitiesTable(containerReport.Name, containerReport.Report),
+			View:  createVulnerabilitiesTable(containerReport.Name, containerReport.Report.Report),
```

Comment:
I know naming is hard, but I don't love `containerReport.Report.Report`!

Reply:
Yeah; we have a few embedded Report structs here. I introduced local variables to make it more readable